templates/kk/core/footer.html

```diff
@@ -1,54 +0,0 @@
-{% load static %}
-{%  load i18n %}
-
-<head>
-    <link rel="stylesheet" href="{% static 'core/css/footer.css' %}">
-</head>
-
-<footer class="container">
-    <hr>
-    <!-- Grid container -->
-    <!-- Section: Social media -->
-    <section class="col text-center">
-    <!-- Facebook -->
-    <a class="btn btn-outline-light btn-floating m-1" href="https://www.facebook.com/kemistklubbenvidaboakademi/" role="button"
-        ><i class="fab fa-facebook-f"></i
-    ></a>
-
-    <!-- Instagram -->
-    <a class="btn btn-outline-light btn-floating m-1" href="https://www.instagram.com/kemistklubben/" role="button"
-        ><i class="fab fa-instagram"></i
-    ></a>
-
-    <!-- Linkedin -->
-    <a class="btn btn-outline-light btn-floating m-1" href="https://www.linkedin.com/company/kemistklubben-vid-%C3%A5bo-akademi-rf/" role="button"
-        ><i class="fab fa-linkedin-in"></i
-    ></a>
-    </section>
-    <!-- Section: Social media -->
-    <div class="row align-middle align-items-center">
-        <div class="col-4 text-center ">
-            <img class="footer-img img-fluid" alt="Axel" src="{% static 'core/images/axel.png' %}">
-        </div>
-        <div class="col-4">
-            <p>
-                {% trans 'Kemistklubben vid Åbo Akademi rf' %}<br>
-                Aurum
-                <br>
-                Henriksgatan 2
-                <br>
-                20500 Åbo
-                <br>
-                Kanslitid Må - To 11:30-13
```

Comment:
KK needs their own footer template because they have their "Kanslitid". Maybe it can be a content variable, but maybe too much work to make the common footer dynamic for this if only KK is using it?